Cargo.toml

```diff
@@ -156,3 +156,12 @@ hotshot-testing ={ git = "https://www.github.com/EspressoSystems/HotShot.git", t
 hotshot-types = { git = "https://www.github.com/EspressoSystems/HotShot.git", tag = "0.5.75-patch1" }
 libp2p-networking = { git = "https://github.com/EspressoSystems//HotShot.git", tag = "0.5.75-patch1" }
 hotshot-example-types = { git = "https://www.github.com/EspressoSystems/HotShot.git", tag = "0.5.75-patch1" }
+
+[profile.dev]
+opt-level = 1
```

Comment:
Is there a reason not to set this (and the other one) to just `opt-level = 0`?

Reply:
I don't remember why I set it to 1.

f329bdf1e61d7729d06d48934265b7957b46edaa